esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -402,9 +402,6 @@ define([
                         if (response.DFUQueryResponse.Warning) {
                             context.filter.open();
                             context.filter.setFilterMessage(context.i18n.FilesWarning);
-                            on(document, "click", function () {
```

Comment:
If this is not here you have the message still appearing if switching widgets.

Reply:
May need to make it on(filter, "unfocus", function() {

}
type of thing?

Reply:
Let me reproduce that...